### PR TITLE
Templates for Pull requests and Issues

### LINKED ISSUE #2674: Issue and Pull Request Templates
it could help to use Issue and Pull Request Templates; see #2652 

Here a some examples for bnd

Issue:
https://github.com/stbischof/bnd/issues/new/choose

Pull Request Templates:
https://github.com/stbischof/bnd/compare/master...stbischof:next

### PR DESCRIPTION
**Pull request by stbischof**
**Status**
- [ ] IN DEVELOPMENT
- [x] READY
- [ ] HOLD

**Description**
it could help to use Issue and Pull Request Templates; see #2652

pull_request_template

issues templates
- bug
- feature
- question

**Checklist**
- [ ] You have added unit tests
- [ ] Documentation
- [x] squash your commits in seperate `git rebase -i` and `git push -f`
- [x] Tag commit-message [exporter,wab,cli,libg,...]
- [x] Sign your work `Signed-off-by: Joe Smith <joe.smith@email.com>` or use `git commit -s`
- [ ] Related Spec: none
- [x] Reference Issues: Fixes #2674 
- [ ] Backward compatibility conflicts:
 * none
- [ ] Known open Issues:
 * none

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+---
+
+**Envorioment (please complete the following information):**
+ - bnd-Version: [e.g. 4.1]
+ - bnd-Part: [e.g. bnd-lib, bnd-cli, gradle, maven, ant, libg]
+ - OS: [e.g. Linux, Windows, iOS]
+ - Java-Version: [e.g. openjdk-11]
+ - OSGi-Framework: [e.g. Felix, Equinox]
+ - Related Spec: [e.g. 112.3.4  Declarative Services - Constructor Injection ]
```
Comment (bjhargrave) [inline .github/ISSUE_TEMPLATE/bug_report.md]: You should also mention the `Git-*` headers. See https://github.com/bndtools/bnd/blob/master/CONTRIBUTING.md#reporting-issues

Comment (stbischof) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+**Envorioment (please complete the following information):**
```
Comment (bjhargrave) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Spelling error: Environment.

**Comment by stbischof** [comment]
I thought this was merged.  Sorry, I've overridden it with an other fix.

**Comment by bjhargrave** [comment]
@bndtools/bnd-hackers Any concerns with adding these issue and PR templates to the repo?

**Comment by bjhargrave** [comment]
The Bnd devs don't think we need this level of process on issue and PR templates at this time. Thanks for the suggestion but for now, we say no thanks to this.

**Pull request closed without merging**